<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aflak</string>
  </property>
  <widget class="AstroImageView" name="astroImageView"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>34</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuFITS">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>FITS</string>
    </property>
    <addaction name="actionSee_FITS_header"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFITS"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open FITS</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSee_FITS_header">
   <property name="text">
    <string>See FITS header</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QWidget</extends>
   <header>pyqtgraph.ImageView</header>
  </customwidget>
  <customwidget>
   <class>AstroImageView</class>
   <extends>ImageView</extends>
   <header>.AstroImageView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
